pages/api/account/manage/profile.js

```diff
@@ -57,7 +57,9 @@ export async function updateProfileApi(username, data) {
     layout: data.layout,
     name: data.name,
     bio: data.bio,
-    tags: data.tags.filter((tag) => tag !== ""),
+    tags: data.tags
+      .filter((tag) => Boolean(tag.trim()))
```

Comment:
Not sure what this is for?

Reply:
Are you talking about Array.filter(item => Boolean(item))?
This is filter technique, which can filter out any falsy values from an array. Values like null, undefined, false , "" etc which are falsy in Javascript, they will be removed from array.

Reply:
Prevoius code was not handling cases for strings like ```"     "```, which contains whitespace.

Reply:
ok thank you! Please don't resolve in the future, otherwise I might miss your replies